bundles/org.openhab.binding.bluetooth.enoceanble/README.md

```diff
@@ -0,0 +1,60 @@
+# EnOcean BLE Binding
+
+This binding adds support for the
+[EnOcean BLE PTM215B](https://www.enocean.com/de/produkte/enocean_module_24ghz_ble/ptm-215b/) rocker.
+
+Currently, it does not work seamlessly work with the
+[BlueZ Binding](https://www.openhab.org/addons/bindings/bluetooth.bluez) because of limitations of this binding,
+particularly the underlying tinyB library. The recommended / tested Bluetooth bridge binding is the
```

Comment:
The tinyB library is no longer used for the BlueZ binding. Have you tested this with the latest version?

Reply:
No, was not aware about it. Will test it.

Reply:
... but it can take some days / weeks because I have to observe long-term performance

Reply:
Then at the very least remove the mention of the tinyB library from this comment.

Reply:
Did do a quick test now, could not get it to work. Adapter status is online but no rocker events are coming in. Restarting openHAB did not change anything. Tested it with the RPI4 integrated adapter as well as a USB adapter that I know is working well for BLE. Log shows no errors or warnings. I could do a deep-dive maybe next weekend. For now I'll just remove the tinyB hint.